mix.exs

```diff
@@ -25,6 +25,7 @@ defmodule OMG.Umbrella.MixProject do
 
   defp deps do
     [
+      {:eleveldb, git: "https://github.com/basho/eleveldb.git", branch: "develop", manager: :make, override: true},
```

Comment:
two comments here:
1/ give a comment why this is overriden :)
2/ can we override to sth less volatile (tag perhaps? a stable branch? - `develop` seems like we could easily go into a broken set of dependencies, if one accidentally upgrades `mix.lock`)

Reply:
I'm afraid that I could only refer to the exact commit, as the fix on `eleveldb` side was applied no sooner than 20 days ago and there is none tag including it.

Reply:
hm, would there be a way to solve this mojave-specific problem with less possibility to backfire to non-mojave users? :thinking: (I just saw Ino's comment - maybe this?)

Reply:
I second  @pdobacz comment. This might be a bit wild :)